apps/voting/app/src/App.js

```diff
@@ -244,9 +245,16 @@ class App extends React.Component {
                   </Title>
                 }
                 endContent={
-                  <Button mode="strong" onClick={this.handleCreateVoteOpen}>
-                    New Vote
-                  </Button>
+                  <React.Fragment>
+                    <BreakPoint to="medium">
+                      <NewVoteButton onClick={this.handleCreateVoteOpen} />
```

Comment:
Same remark [than on the Token Manager](https://github.com/aragon/aragon-apps/pull/612/files#r248775882) about this button name: shall we change it, or have an icon mode + a label mode in the button itself?